dev/benchmarks/test_apps/stocks/l10n.yaml

```diff
@@ -16,3 +16,4 @@ output-localization-file: stock_strings.dart
 ## generating Flutter's localization files.
 synthetic-package: false
 template-arb-file: stocks_en.arb
+nullable-getter: false
```

Comment:
almost all other options here are documented. Should this one get some documentation as well?